test_files/super.tr.js

```diff
@@ -59,3 +59,6 @@ class SuperTestDerivedInterface {
         // Sickle: end stub declarations.
     }
 }
+class SuperTestStaticProp extends SuperTestBaseOneArg {
+}
```

Comment:
This bit shows why this change is necessary.  It's important here that we don't create a synthetic ctor, because we don't know the argument to pass to the base class.